Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_aitdm.nut

```diff
@@ -1,22 +1,36 @@
 untyped
 global function GamemodeAITdm_Init
 
-const SQUADS_PER_TEAM = 3
+// these are now default settings
+const int SQUADS_PER_TEAM = 4 // was 3
```

Comment:
No need to save old value in comment. That's what commit history and `git blame` is for :P
```suggestion
const int SQUADS_PER_TEAM = 4
```